jupyter-extension/jupyterlab_pachyderm/mount_server_client.py

```diff
@@ -37,8 +46,11 @@ def __init__(
             self.sock_path = ""
         self.client = AsyncHTTPClient()
         # non-prived container flag (set via -e NONPRIV_CONTAINER=1)
-        # TODO: Would be preferable to auto-detect this, but unclear how
+        # or use DET_RESOURCES_TYPE environment variable to auto-detect this.
         self.nopriv = NONPRIV_CONTAINER
+        self.determined_resources_type = DET_RESOURCES_TYPE
```

Comment:
What is the need for making an additional variable that is never referenced?

Why not just:

`self.nopriv = NONPRIV_CONTAINER or DET_RESOURCES_TYPE == SLURM_JOB`